openjdk/CompileThirdPartyHeap.gmk

```diff
@@ -10,6 +10,9 @@ MMTK_CPP_ROOT = $(THIRD_PARTY_HEAP)
 OPENJDK_VERSION=`cd $(MMTK_RUST_ROOT) ; cargo read-manifest --manifest-path=Cargo.toml | python3 -c 'import json,sys; print(json.load(sys.stdin)["metadata"]["openjdk"]["openjdk_version"])'`
 OPENJDK_LOCAL_VERSION=`git rev-parse HEAD`
 
+# Get the current host triple. This is used for the PGO build
```

Comment:
The change looks fine. But this seems to be a general change. How is this related with PGO?

Reply:
I'm following what is mentioned in the Cargo PGO build guide: https://doc.rust-lang.org/rustc/profile-guided-optimization.html#a-complete-cargo-workflow. Specifically:
> We pass the --target flag to Cargo, which prevents the RUSTFLAGS arguments to be passed to Cargo build scripts. We don't want the build scripts to generate a bunch of .profraw files.

Reply:
I see. Probably add the link in the comment so we won't get confused in the future (or attempt to remove that).